build.assets/Dockerfile-grpcbox

```diff
@@ -0,0 +1,44 @@
+FROM golang:1.20
+
+# Image layers go from less likely to most likely to change.
+RUN apt-get update && \
+  apt-get install -y --no-install-recommends \
+    npm \
+    unzip \
+    && \
+  rm -rf /var/lib/apt/lists/*
+
+# protoc-gen-gogofast
+# Keep in sync with api/proto/buf.yaml (and buf.lock)
+ARG GOGO_PROTO_TAG # eg, "v1.3.2"
+RUN go install "github.com/gogo/protobuf/protoc-gen-gogofast@$GOGO_PROTO_TAG"
+
+# protoc-gen-js and protoc-gen-ts
+ARG NODE_GRPC_TOOLS_VERSION # eg, "1.12.4"
+ARG NODE_PROTOC_TS_VERSION  # eg, "5.0.1"
+RUN npm install --global "grpc-tools@$NODE_GRPC_TOOLS_VERSION" "grpc_tools_node_protoc_ts@$NODE_PROTOC_TS_VERSION"
+
+# protoc
+ARG PROTOC_VER # eg, "3.20.2"
+RUN VERSION="$PROTOC_VER" && \
+  PB_REL='https://github.com/protocolbuffers/protobuf/releases' && \
+  PB_FILE="$(mktemp protoc-XXXXXX.zip)" && \
+  curl -fsSL -o "$PB_FILE" "$PB_REL/download/v$VERSION/protoc-$VERSION-linux-$(uname -m).zip"  && \
```

Comment:
When running this on M1, I get this error:

```
 > [5/8] RUN VERSION="3.20.2" &&   PB_REL='https://github.com/protocolbuffers/protobuf/releases' &&   PB_FILE="$(mktemp protoc-XXXXXX.zip)" &&   curl -fsSL -o "$PB_FILE" "$PB_REL/download/v$VERSION/protoc-$VERSION-linux-$(uname -m).zip"  &&   unzip "$PB_FILE" -d /usr/local &&   rm -f "$PB_FILE":
#0 0.599 curl: (22) The requested URL returned error: 404
```

I think that's because `uname -m` returns arm64 while protoc releases use aarch_64. After replacing that subshell with hardcoded aarch_64 `make grpc` went through with no problems.

Reply:
https://github.com/gravitational/teleport/pull/26758